website/src/data/users.tsx

```diff
@@ -205,6 +205,15 @@ const Users: User[] = [
     source: 'https://github.com/ipavlic/apex-fp/tree/master/website',
     tags: ['opensource'],
   },
+  {
+    title: 'Astronomer',
+    description:
+      'Enterprise-grade framework for Apache Airflow. Production-ready Airflow environments with just a few clicks',
+    preview: require('./showcase/astronomer.png'),
+    website: 'https://docs.astronomer.io',
+    source: 'https://github.com/astronomer/docs',
+    tags: ['product', 'versioning'],
```

Comment:
```suggestion
    tags: ['product', 'versioning', 'opensource'],
```

Missing opensource tag ;)